ee/codegen/src/generators/prompt-block.ts

```diff
@@ -189,11 +189,14 @@ export class PromptBlock extends BasePromptBlock<PromptBlockType> {
     classArgs.push(
```

Comment:
can make this easier by:
1. defining `const nonEmpty = promptBlock.text === "";`
2. Replace all the `true` statments with just setting to `nonEmpty`

Do this for the file below as well